service/controller/v1/key/key.go

```diff
@@ -21,24 +19,8 @@ const (
 	ServiceTypeManaged = "managed"
 )
 
-func OperatorChannelName(customResource v1alpha1.Release) string {
```

Comment:
Lots of information are based on the "authorities" now and I am thinking that we should move more key functions to `apiextensions` in general.

Reply:
Moving most of these functions to apiextensions would make sense to me.